### ISSUE TITLE
Issue about image format?

### ISSUE THREAD

**Issue by YongboLiang** [MIVRC/MSRN-PyTorch#5]
Hi, thanks for your sharing. It's a wonderful job.
The training dataset is DIV2K in your paper and the images are PNG format. However, the image format of testing dataset are BMP and JPG. 
I found that BMP images could be reconstructed well while the reconstruction of JPG images were poor when the training dataset was original DIV2K. 
Then I want to know how do you realize  reconstruction quality of JPG images.  Do you make JPG
training dataset with DIV2K?
Best

**Comment by CV-JunchengLi** [comment]
Hi, I noticed your issue.
You are right. The reconstruction of JPEG images was poor when the training dataset was PNG format.
The correct approach may be used JPEG-input / PNG-output as training data to fine-tuning the SR model.
However, we did not consider this issue at the time.
In our new code, we use the code provided by the RCAN author to preprocess the test image.
All test datasets are processed into png format.
More information can be found in MSRN-PyTorch/MSRN/Test/Prepare_TestData_HR_LR.m
I hope this will help you.
